static/js/components/JobDetails.jsx

```diff
@@ -190,8 +227,9 @@ var JobDetails = React.createClass({
   },
 
   cronUsage: function() {
+    var msg = "Note: To reduce errors, direct modifications aren't currently supported. Please instead deactivate the old job and add a new.";
```

Comment:
I think this should also note the interval at which jobs are updated in the database